cts/CIB.py

```diff
@@ -353,10 +353,10 @@ def add_resources(self):
         m.add_op("monitor", "P10S")
         m.commit()
 
-        # Ping the test master
+        # Ping the test main
         p = Resource(self.Factory, "ping-1","ping",  "ocf", "pacemaker")
         p.add_op("monitor", "60s")
-        p["host_list"] = self.CM.Env["cts-master"]
+        p["host_list"] = self.CM.Env["cts-main"]
```

Comment:
For the purposes of this code, "exerciser" would be a better replacement than "main", since that's how it's referred to in the documentation (cts/README.md). Also, please update the copyright year at the top of the file.